README.md

```diff
@@ -7,6 +7,7 @@ This program is written in Python. Its purpose is to collect weather forecast da
 
 You need to register yourself on few websites (list below) to receive an API key:
 - https://www.weatherapi.com/signup.aspx
+- https://rapidapi.com/meteostat/api/meteostat/pricing #plan basic
```

Comment:
These links are different, one redirects to a login page, and the second to a pricing plan. Also, you talk about API key obtainment but neither the text here nor the links are not telling how to achieve that. Please make sure to include that information as well.

Reply:
Sorry for the confusion, but when you are already registered some of the websites look different. And to be honest registration and finding the API key on the second webpage is not so easy. I hope that after additional info it will be easier.

Reply:
Thanks for adding that additional info to the README file, though I'd recommend updating those new links to point to those APIs documentation, however exact they are in this regard:

- for weatherapi: https://www.weatherapi.com/docs/#intro-getting-started 
- for rapidapi: https://docs.rapidapi.com/docs/keys-and-key-rotation#how-to-find-your-api-key